tox.ini

```diff
@@ -12,20 +12,29 @@ commands =
     coverage report
     coverage xml
 
+[testenv:isal]
+deps =
+    pytest
+    coverage
+    isal
+
 [testenv:flake8]
 basepython = python3.6
 deps = flake8
 commands = flake8 src/ tests/
+skip_install = true
 
 [testenv:mypy]
 basepython = python3.6
 deps = mypy
-commands = mypy src/
+commands = mypy --ignore-missing-imports src/
+skip_install = true
 
 [flake8]
 max-line-length = 99
 max-complexity = 10
 extend_ignore = E731
+skip_install = true
```

Comment:
I think this line should not be added (this section is only read by `flake8`).

Reply:
Done